tool/ci.dart

```diff
@@ -59,12 +63,7 @@ ArgParser makeArgParser() {
       help: 'Disable all tasks. Overridden by --<task> flags.',
```

Comment:
Can't add comments above this, thanks GitHub!

It's probably useless to have the `none` flag now so you can remove it.

Reply:
Good point! 🤓  Thanks @HosseinYousefi !